### PR TITLE
Add JS-Tiny-World

### PR DESCRIPTION
**Pull request by bugagashinka**
### 7. A Tiny JS World

[Demo](https://bugagashinka.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/bugagashinka/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/bugagashinka/js-tiny-world/index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+const inhabitants = [dog, cat, woman, man];
+
+woman.friends = [cat, man];
+man.friends = [woman, dog];
+dog.friends = [man];
+cat.friends = [woman];
+
+// ======== OUTPUT ========
+inhabitants.forEach(inhabitant => {
+  const info = Object.keys(inhabitant).map(prop => {
```
Comment (OleksiyRudenko) [inline submissions/bugagashinka/js-tiny-world/index.js]: The order of properties in an object is not guaranteed.
Please, refer to props explicitly. You may e.g. list prop values in an array that will be then further processed.

File: submissions/bugagashinka/js-tiny-world/index.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+const catWoman = {
+  species: 'human',
+  name: 'Kitty woman',
+  gender: 'female',
+  legs: 0,
+  hands: 2,
+  get saying() {
+    return cat.saying;
+  },
+};
```
Comment (OleksiyRudenko) [inline submissions/bugagashinka/js-tiny-world/index.js]: Making a Cat Woman is optional. However, you did that. Is there any reason to not to list the `catWoman` among inhabitants?

Comment (bugagashinka) [reply submissions/bugagashinka/js-tiny-world/index.js]: I think no, fix this omission

File: submissions/bugagashinka/js-tiny-world/index.js (hunk 1 above)
Comment (bugagashinka) [reply submissions/bugagashinka/js-tiny-world/index.js]: Thanks for this improvement

File: submissions/bugagashinka/js-tiny-world/index.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+woman.friends = [cat, man, catWoman];
+man.friends = [woman, dog];
+dog.friends = [man];
+cat.friends = [woman, catWoman];
+catWoman.friends = [woman, cat];
+
+// ======== OUTPUT ========
+inhabitants.forEach(inhabitant => {
+  const info = props.map(prop => {
+    if (prop === 'friends') {
```
Comment (OleksiyRudenko) [inline submissions/bugagashinka/js-tiny-world/index.js]: This is the good place to employ ternary operator.
Please, do.

Comment (bugagashinka) [reply submissions/bugagashinka/js-tiny-world/index.js]: Done

**Pull request merged**